util/fipstools/acvp/modulewrapper/modulewrapper.cc

```diff
@@ -1701,7 +1729,9 @@ static bool RSAKeyGen(const Span<const uint8_t> args[], ReplyCallback write_repl
     return false;
   }
 
-  CachedRSAKeys().emplace(static_cast<unsigned>(bits), std::move(key));
+  if (AddRSAKeyToCache(key, bits) == nullptr) {
```

Comment:
I noticed that the naming of all RSA pointer variables passed into `AddRSAKeyToCache` were all changed to `rsa`, should we change this one to `rsa` as well?

Reply:
Addressed.